### PR TITLE
doc suggestions

### PR DESCRIPTION
**Pull request by paddymul**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,9 +1,15 @@
 # Contra
-> The Contra utility converts the so called "C Style Markup Language" to HTML/XML and back.
+> The Contra utility converts [C Style Markup Language](definition of C style markup) to HTML/XML and back.
```
Comment (paddymul) [inline README.md:2]: Is the C Style markup language something that you created?  Is there a formal project page for it?  Link to that here.  Or if it's your own creation, link to the docs for it.

**Comment by paddymul** [comment]
Especially for little known projects that aren't established, your README is your chance to market the project.

You want to quickly establish what your project does in a sentence or two.

Next you want to describe what problem your package solves.

Then show an example.

Then explain how to use it.  I personally don't care about how to use a project (I don't want to invest the time) until I understand that it is solving a valuable problem for me.

This changes a bit if your project is more popular and/or other projects depend on it.  Then people could be visiting your project because they need to understand it in the context  of some other tool

File: README.md [hunk 2]
```diff
@@ -33,7 +39,20 @@ then to type out this with:
 
 Yet, the two are equivalent using Contra.
```
Comment (paddymul) [inline README.md:40]: It's not quite homoiconicity, but the circularity is important.  Describe how this is useful (I know it is, I just can't come up with that right now.

**Comment by agvxov** [comment]
Ok, thats all fair, thanks.
Do you think d11cd9f4ea9683636a7b455a32f8b7336044a5d7 is an improvement?

**Pull request closed without merging**